src/components/ContactsList/_modals/ContactAdd.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC, useState } from 'react'
+import { Button } from '@/components/Button'
+import { ContactAddModal } from './ContactAddModal/ContactAddModal'
+import { useContacts } from '@/providers'
+
+export const ContactAdd: FC = () => {
+  const [open, setOpen] = useState(false)
+  const { refresh } = useContacts()
+
+  return (
+    <>
+      <Button variant="primary" small onClick={() => setOpen(true)}>
+        Add contact
+      </Button>
+      {open && (
+        <ContactAddModal
+          open={true}
+          onSubmitted={() => {
+            setOpen(false)
+            refresh()
```

Comment:
If we have more than one function in handler it's nice to create a separate handler function.

`const handleSubmit = () => {
setOpen(false);
refresh();
}`

`onSubmitted={handleSubmit}`

Reply:
Yes. Will add `onSubmitted` func